src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: "
+                + String.format("%.1f",getArea()) + " sq. units, radius: "
+                + radius + " units, color: " + getColor();
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/63095184/11593123-285d-426f-938c-bc40cb1c8da6)
You need method draw() use interface for it